app/Http/Controllers/QuestionApiController.php

```diff
@@ -32,4 +32,24 @@ public function delete(Question $question): JsonResponse
 
         return response()->json(['message' => 'Question deleted']);
     }
+
+    public function update(CreateQuestionRequest $request, Question $question): JsonResponse
+    {
+        $question->question = $request->question;
+        $question->points = $request->points;
+        $question->hint = $request->hint;
+        $question->quiz_id = $request->quiz_id;
```

Comment:
The quiz_id shouldn't be editable. 

You can still re-use your Request object for this, in the rules method you can check what the method is with:


```
 if ($this->method() === 'POST') {

}
```

So you could do something like this in the rules method of the request object:

```
$rules = [rules for update here];

 if ($this->method() === 'POST') {
    $rules['quiz_id'] = 'whatever rules you need for quiz_id in create';
}

return $rules;
```